Assets/scripts/Attack.cs

```diff
@@ -4,14 +4,55 @@
 
 public class Attack : MonoBehaviour
 {
+    List<GameObject> enemys;
+    GameObject enemy;
+
+    float shortDis;
+
+    Vector3 attackPos;
+
+    void Start()
+    {
+        enemys = new List<GameObject>(GameObject.FindGameObjectsWithTag("Enemy"));
+
+        // 여기 내용 공부할것
+        if (enemys.Count > 0)
+        {
+            //Distance 거리 재기?
+            shortDis = Vector3.Distance(gameObject.transform.position, enemys[0].transform.position);
+            enemy = enemys[0];
+
+            foreach (GameObject foundEnemy in enemys)
+            {
+                float Distance = Vector3.Distance(gameObject.transform.position, foundEnemy.transform.position);
```

Comment:
```suggestion
                float distance = Vector3.Distance(gameObject.transform.position, foundEnemy.transform.position);
```
변수명은 앞글자 소문자로 규칙 맞처두면 좋을듯

Reply:
고쳤숨다